app/models/person_upload.rb

```diff
@@ -0,0 +1,32 @@
+require 'virtus'
+class PersonUpload
```

Comment:
Oh hey, is there a spec for this class somewhere? If not what's the thinking behind omitting it?